detectron2/data/dataset_mapper.py

```diff
@@ -84,15 +84,16 @@ def __call__(self, dataset_dict):
         else:
             # Crop around an instance if there are instances in the image.
             # USER: Remove if you don't use cropping
-            if self.crop_gen:
+            crop_around_instance = self.crop_gen and dataset_dict["annotations"]
```

Comment:
it's easier to just replace L80 by `if dataset_dict.get("annotations", []):`, to still enable cropping.

Reply:
:+1: fixed